lib/ansible/plugins/doc_fragments/zabbix.py

```diff
@@ -48,4 +48,7 @@ class ModuleDocFragment(object):
       type: bool
       default: yes
       version_added: "2.5"
+notes:
+    - Beware that using I(login_password=zabbix) will result in word "zabbix" being replaced by "********" in every module output as I(login_password) content
```

Comment:
How about adding a link and a reference to `no_log` so folks who want more information can find it more easily?
```suggestion
    - If you use I(login_password=zabbix), the word "zabbix" is replaced by "********" in all module output,
      because I(login_password) uses C(no_log) to prevent leaks of secret information. See L(this FAQ,https://docs.ansible.com/ansible/devel/network/user_guide/faq.html#why-is-my-output-sometimes-replaced-with) for more information.
```